components/editor/TimesheetEntryEditor.tsx

```diff
@@ -339,12 +355,30 @@ export const TimesheetEntryEditor = ({
                 onActiveStartDateChange={onYearOrMonthChange}
                 value={datesValue(dates)}
                 tileClassName={({ date: thisDate }) => {
+                    let className = ""
+                    if (matchDates(thisDate)) {
+                        className +=
+                            "react-calendar__month-view__public_holidays "
+                    }
                     if (entryDates.includes(jsDateToShortISODate(thisDate))) {
-                        return "react-calendar__tile--completed"
+                        className += "react-calendar__tile--completed"
                     }
-                    return ""
+                    return className.trim()
```

Comment:
This can be done using an immutable list instead of a mutable string with the spaces between classnames
```ts
const classNames = []
if (matchDates(thisDate)) {
  classNames.push("react-calendar__month-view__public_holidays")
}
if (entryDates.includes(jsDateToShortISODate(thisDate))) {
  classNames.push("react-calendar__tile--completed")
}
return classNames.join(" ")
```